www/js/share/components/check-user-status/check-user-status.controller.js

```diff
@@ -179,19 +178,10 @@
             }
         };
 
-        vm.authenticateStatus = function(status) {
-            vm.authenticated = status.indexOf("authenticated") > -1;
-            if (
-                !vm.authenticated &&
-                (vm.isFinancial ||
-                    (vm.isCR && (vm.balance >= 200 || vm.mt5LoginList.length > 0)) ||
-                    vm.isMLT ||
-                    vm.isMX)
-            ) {
-                if (!appStateService.hasAuthenticateMessage) {
```

Comment:
@nazaninreihani 
Do you handle that this part of code is execute just one time for each user authentication process?

As you know if our app try to keep itself connect, so it is possible to have multiple authentication process for an account during the runtime and in this way you push same messages in your messages list.

I think you should keep this condition in your new code, unless you handle it some where else.